tests/config.go

```diff
@@ -31,8 +31,6 @@ import (
 type KubeVirtTestsConfiguration struct {
 	// StorageClass to use to create local PVCs
 	StorageClassLocal string `json:"storageClassLocal"`
-	// StorageClassHostPathSeparateDevice to use to create host-path PVCs that are on a separate device from the boot device
```

Comment:
So want to keep this particular one. It tests a very specific case of having a hostpath PV on a separate device that is used when hotplugging a volume. We need some specific setup for that and there is no generic storage capability we can replace it with.

Reply:
I just removed it from the config file and created that storageclass manually from the tests that using this storageclass, I saw no reason for it to be configurable

Reply:
Fair enough, I just don't see it in this PR.

Reply:
Okay, well I should look better, just noticed all the changes are in util.go